explorer/websocket.go

```diff
@@ -294,23 +301,37 @@ func (wsh *WebsocketHub) run() {
 				wsh.bufferMtx.Unlock()
 				continue
 			}
+
+			// Copy the transaction slice and make a new empty buffer.
 			txs := make([]*types.MempoolTx, len(wsh.newTxBuffer))
 			copy(txs, wsh.newTxBuffer)
 			wsh.newTxBuffer = make([]*types.MempoolTx, 0, newTxBufferSize)
 			wsh.bufferMtx.Unlock()
+
 			if len(wsh.clients) > 0 {
 				log.Debugf("Signaling %d new tx to %d clients", len(txs), len(wsh.clients))
 			}
-			for signal, client := range wsh.clients {
+			for clientSpoke, client := range wsh.clients {
+				// Each client gets the same tx slice. In the future each client
+				// may have a different slice of new transactions.
 				client.cl.Lock()
 				client.cl.newTxs = txs
 				client.cl.Unlock()
+
+				// Inform the client's websocket connection handler
+				// (RootWebsocket) of the new transactions, but send a nil slice
+				// in the message since the client accesses the tx slice stored
+				// in its newTxs field.
 				select {
-				case *signal <- pstypes.HubMessage{Signal: sigNewTx}:
+				case *clientSpoke <- pstypes.HubMessage{
+					Signal: sigNewTxs,
+					Msg:    ([]*types.MempoolTx)(nil), // client has the data in its newTxs field
```

Comment:
Give the `RootWebsocket` http handler (upgraded to websocket) a valid `SigNewTxs` event.  This was a bug.